### PR TITLE
remove start and end date from item and fix ts errors

### PR DESCRIPTION
**Pull request by hobbitronics**
## Removed
- start and end date from ItemForm

## Added
- make check
- make lint

## Fixed
- broken import
- lots of ts errors/warning (down to 5 mostly from bad routify types)

### CODE REVIEW

File: src/components/FilePreview/FilePreview.svelte [hunk 1]
```diff
@@ -31,7 +31,7 @@ function onDelete(event: CustomEvent, id: string) {
 }
 </style>
 
-<div class="preview flex justify-between align-items-center br-8px p-10px mb-1 {$$props.class}" on:click>
+<div class="preview flex justify-between align-items-center br-8px p-10px mb-1 {$$props.class}" on:click on:keydown={(e) => {if (e.key === 'Enter') e.currentTarget.click()}}>
```
Comment (briskt) [inline src/components/FilePreview/FilePreview.svelte:34]: Doesn't look related to "remove start and end date from item". Can you add something to the PR description and/or move it to its own PR?

**Review by briskt — APPROVED** [review]
Thanks for bringing this up to date and cleaning up the types. Can you add to the PR title something about Typescript cleanup? Better to use separate PRs, but this is ok for now.

File: src/components/FilePreview/FilePreview.svelte (hunk 1 above)
Comment (hobbitronics) [reply src/components/FilePreview/FilePreview.svelte:34]: Sorry, I wanted to do ts checks to make sure this change didn't cause any other issues and got carried away.

Comment (briskt) [reply src/components/FilePreview/FilePreview.svelte:34]: It looked to me like this added handling for Enter keystroke. Is that correct?

Comment (hobbitronics) [reply src/components/FilePreview/FilePreview.svelte:34]: yes, there was a ts warning about not having non interactive elements with click handler, but no key handler.
